scipy/signal/tests/test_spectral.py

```diff
@@ -1085,7 +1085,7 @@ def test_input_validation(self):
         assert_raises(ValueError, check_NOLA, 'hann', 64, -32)
 
         x = np.zeros(1024)
-        z = np.array(stft(x), dtype=object)
+        z = stft(x)[2]
 
         assert_raises(ValueError, stft, x, window=np.ones((2,2)))
```

Comment:
Other tests also should use `with assert_raises(ValueError, match=msg):` instead of `assert_raises`?

Reply:
Fair point - I modified all other ValueError tests in `TestSTFT.test_input_validation()`  in commit 471e01a. It seems like I messed a bit up while rebasing onto current main - I didn't mean to pull in all the other commits...